core/templates/components/common-layout-directives/common-elements/sharing-links.component.html

```diff
@@ -68,21 +68,11 @@
   }
 
   .oppia-sharing-links .share-option-img {
-    border-radius: 5px;
+    border-radius: 3.5px;
```

Comment:
Will it be possible to use `em` or `rem` values here? We are planning to use relative units in the future (see https://github.com/oppia/oppia/issues/10754).

Reply:
okay

Reply:
Should I change the styling from px to rem/em for the entire sharing-links and info modal card?  It will take little time but I guess will be helpful if we are looking forward for #10754.